pkg/sbom/spdx/unmarshal.go

```diff
@@ -70,6 +73,11 @@ func (s *SPDX) unmarshal(spdxDocument *spdx.Document) error {
 	apps := map[common.ElementID]*ftypes.Application{}
 	packageSPDXIdentifierMap := createPackageSPDXIdentifierMap(spdxDocument.Packages)
 
+	relationships := lo.Filter(spdxDocument.Relationships, func(rel *v2_3.Relationship, _ int) bool {
```

Comment:
```suggestion
	relationships := lo.Filter(spdxDocument.Relationships, func(rel *spdx.Relationship, _ int) bool {
```

this does not work?

Reply:
Done
https://github.com/aquasecurity/trivy/pull/4210/commits/9e237b9883785a9b0a8eae8f3a2e61e0b8811486